### PR TITLE
Vendor xterm dependencies in atlantis 

### PR DESCRIPTION
**Pull request by Aayyush**
This avoids downtime when the CDN hosting the xterm libraries is not available

### CODE REVIEW

**Review by nishkrishnan — COMMENTED** [review]
Tbh I've never done this before with javascript, probably good to get someone else to review on the team  cc: @mikecutalo @danielhochman

File: server/static/bindata_assetfs.go [hunk 1]
```diff
@@ -179,47 +183,127 @@ func staticImagesAtlantisIcon_512Png() (*asset, error) {
...
...\xaa\x7b\xe1\xdc\xb3\x69\x99\x57\xc5\x9a\xbd\xe1\x09\xbb\x9f\x08\xf7\x4e\xe2\x82\xc2\x6c\xe8\x02\xf0\x2c\x56\xdd\xe1\x94\x4f\x25\xa1\xba\x4a\xaf\xb3\x94\xdf\x80\xc8\xd3\x39\xb4\x4d\xe6\x56\xc6\xb1\x9c\x07\x93\x79\xdb\xcb\x44\x4e\xe7\xa1\x07\x0b\x8e\xca\x0f\xd8\xab\xc7\x48\xa6\xec\x30\xa8\x62\x28\x15\xce\x5c\x6e\x99\xa3\x7a\x38\x96\x86\x9f\xaa\x4d\x77\x13\xb5\xed\x8a\xba\xf6\x14\x37\x07\x77\x6e\x7b\x2e\xa4\xdd\xfc\xc9\x38\x36\x79\xb1\x03\x05\xd5\x92\x75\x00\x64\x34\xef\xf0\x09\x4b\x2f\x8b\xaf\x59\xa6\x4a\x3a\xd7\xce\x37\x9d\x0a\xec\x87\xb2\x6f\xc1\xd1\x6d\x5a\xbe\x74\x1e\xd4\xb5\xfb\x64\x44\xe9\x48\xf8\x7e\x2c\xb7\xc0\xd0\xd7\x4e\xeb\x72\xcc\xee\x3b\x67\xdc\xb7\x0c\xc5\x76\xdc\x99\xc3\x09\xe5\xf6\x69\x4e\xc7\x39\x71\x5f\x75\xa4\x8e\x31\xa8\x93\x0d\xbf\x90\x63\x92\xd2\xa2\x0b\xea\xe9\x64\x82\x59\xc8\x69\x11\xa6\x91\x24\x05\x20\x10\x18\x21\x21\x7f\xe4\x35\xc6\x8d\xfc\xdf\x76\xe9\xa1\xb3\xe9\x7d\x7f\x48\xd3\x3e\x4c\xf1\x7c\x9f\x35\x9b\xbc\x40\xec\x2c\xe5\x67\x09\x2d\xd9\x54\x4b\x9d\x28\x88\xc4\x4b\x39\x7d\x3f\x7f\xf3\x35\x1d\x84\xa7\x78\xc7\xca\x7d\xbc\x66\x3f\xbe\x7d\x43\x38\x45\x3d\x2e\x45\xd2\x0d\x2b\x8e\xd1\x0d\x1b\x61\xee\x2f\x9a\xdc\xd4\x35\xf7\x7d\x6e\x21\xb3\xae\x3d\x79\xd6\x90
```
Comment (samrabelachew) [inline server/static/bindata_assetfs.go:211]: Is there a reason why we can't read this in from a separate file?

Comment (Aayyush) [reply server/static/bindata_assetfs.go:211]: This is a generated file
```
// Code generated for package static by go-bindata DO NOT EDIT. (@generated)
```

**Review by danielhochman — APPROVED** [review]
lgtm

**Pull request merged**